packaging_automation/tests/test_package_cloud_statistics_collector.py

```diff
@@ -53,7 +53,7 @@ def get_filtered_package_count(session) -> int:
     # Since package count for our test repo is lower than 100, we get the total package details by getting all the
     # packages in one call
     result = stat_get_request(
-        package_list_with_pagination_request_address(PACKAGE_CLOUD_PARAMETERS, 1, 100),
+        package_list_with_pagination_request_address(PACKAGE_CLOUD_PARAMETERS, 1, 200),
```

Comment:
Why did we need to update this?

Reply:
Recently I put new packages into azure repo which exceeds 100 package . I'm using azure repo as my test repo to test the package counts. After exceed, my tests failed since some packages was outside the pagination limit.